apps/ios/Shared/ContentView.swift

```diff
@@ -64,11 +63,11 @@ struct ContentView: View {
     private func runAuthenticate() {
         if !prefPerformLA {
             userAuthorized = true
-        } else if showChatInfo {
-            showChatInfo = false
-            DispatchQueue.main.async {
-                justAuthenticate()
-            }
+//        } else if showChatInfo {
+//            showChatInfo = false
+//            DispatchQueue.main.async {
+//                justAuthenticate()
+//            }
```

Comment:
we're going to have more sheet views opening from chat view for groups - we should look for a less fiddly solution. maybe at least a single binding for all such views affecting them via onChange modifiers